front-app/src/views/Register.tsx

```diff
@@ -85,11 +88,17 @@ const Register = () => {
           />
 
           <TextInput register={register} placeholder='Nombre de su empresa' name='company' />
+
+          <TextInput register={register} placeholder='Imagen de perfil' name='profileImage' />
+
+          <TextInput register={register} placeholder='Puesto en empresa' name='position' />
+
           <Button loading={isLoading} text='Loading...'>
             registrarse
           </Button>
         </form>
       </div>
+      <HelpComponent></HelpComponent>
```

Comment:
Same here, let's move this to the Left layout component and position it absolutely